Datadog/Example/Scenarios/RUM/SwiftUIInstrumentation/SwiftUIRootViewController.swift

```diff
@@ -44,12 +44,12 @@ struct RootView: View {
         // An issue was introduced in iOS 14.2 (FB8907671) which makes
         // `TabView` items to be loaded twice, once when the `TabView`
         // appears` and once when the Tab item itself appears. This
-        // lead to RUM views being reported twice. This issue was fixed
-        // in iOS 14.5, see https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-14_5-release-notes
+        // lead to RUM views being reported twice. This issue seems to
+        // have been fixed in iOS 15.
         // As a workaround, the tab view items can be embedded in a
         // `LazyVStack` or `LazyHStack` to lazily load its content when
         // it needs to render them onscreen.
-        if #available(iOS 14.5, *) {
+        if #available(iOS 15, *) {
```

Comment:
Why exactly we need this change in this PR 🤔💭?

Reply:
Arg, this is not related to this PR scope. The new CI stack has highlighted that the bug was not actually fixed in iOS 14.5 but in 15 (Or maybe in between but we have no simulator to test, xcode provide simulators for iOS 14.5 and the iOS 15). The iOS 14.5 changelog might be referencing another issue..

This change is also in #657, I will rebase when it get merged!